composer.json

```diff
@@ -1,65 +1,74 @@
 {
-    "name": "drupal-composer/drupal-project",
-    "description": "Project template for Drupal 8 projects with composer",
-    "type": "project",
-    "license": "GPL-2.0+",
-    "authors": [
-        {
-            "name": "",
-            "role": ""
-        }
-    ],
-    "repositories": [
-        {
-            "type": "composer",
-            "url": "https://packagist.drupal-composer.org"
-        }
+  "name": "drupal-composer/drupal-project",
```

Comment:
As far as I know , this is obsolete. We should follow what https://www.drupal.org/node/2404989 says. Thoughts @nuez?